packages/toolpad-app/src/utils/usePageTitle.ts

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+
+export default function usePageTitle(title: string) {
```

Comment:
So no Next.js https://nextjs.org/docs/api-reference/next/head 😁.

Reply:
We can use it. At the moment Next.js is mostly used for top-level routing and API routes. Almost all other parts of Toolpad are Next.js agnostic (The editor and the applications use React router, lazy loading uses `React.lazy`,...). It depends on how much we want to lock into their APIs. 
In my opinion, their routing doesn't always work all that well for SPAs.